src/source-pcap.c

```diff
@@ -641,3 +705,223 @@ void PcapTranslateIPToDevice(char *pcap_dev, size_t len)
 
     pcap_freealldevs(alldevsp);
 }
+
+#ifdef UNITTESTS
+
+static uint32_t Upper32(uint64_t value)
+{
+    return value >> 32;
+}
+static uint32_t Lower32(uint64_t value)
+{
+    return value; /* slice lower 32bits */
+}
+
+static int UpdatePcapStatsValue64NoChange01(void)
+{
+    /*
+     * No change in counter values.
+     * Last count is within first 32bit range, i.e. same as pcap_stat range.
+     */
+    uint64_t last[] = { 0, 12345, (uint64_t)UINT32_MAX };
+    u_int current[] = { 0, 12345, UINT_MAX };
```

Comment:
Am using u_int type just to be in line with the actual pcap_stats definition. uint32_t would be the same.